### PR TITLE
report viewer sample

### PR DESCRIPTION
**Pull request by grantbacon-jaspersoft**

### CODE REVIEW

File: react-app-render/src/Main.tsx [hunk 1]
```diff
@@ -1,67 +1,65 @@
...
+        pageName = "inputControls";
+      } else if (lastPartOfUrl.endsWith(inputControlsTypesPageStart)) {
+        pageName = "inputControlsTypes";
+      } else if (lastPartOfUrl.endsWith(schedulerPageStart)) {
+        pageName = "scheduler";
+      } else if (lastPartOfUrl.endsWith(applicationPageStart)) {
+        pageName = "application";
+      } else if (lastPartOfUrl.endsWith("input-controls/report-sample.html")) {
+        setSampleComponent(
+          <ReportSample vContainer={vContainer} uri={resourceUri} />,
```
Comment (ecanchev-jaspersoft) [inline react-app-render/src/Main.tsx:41]: move this to `getLiveSamplePage()` method, to have the same behavior please.

**Review by ecanchev-jaspersoft — CHANGES_REQUESTED** [review]

File: react-app-render/src/Main.tsx (hunk 1 above)
Comment (grantbacon-jaspersoft) [reply react-app-render/src/Main.tsx:41]: I propose we eliminate `getLiveSamplePage` from all samples, as it adds unneeded complexity to our samples by adding a new layer of mapping from URL -> arbitrary name -> sample component

Comment (ecanchev-jaspersoft) [reply react-app-render/src/Main.tsx:41]: ok sounds good, pls remove it for all options.

**Review by ecanchev-jaspersoft — APPROVED** [review]
approved

**Pull request merged**